project/pom.xml

```diff
@@ -365,6 +365,12 @@
       <artifactId>assertj-guava</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.jcraft</groupId>
+      <artifactId>jsch</artifactId>
+      <version>0.1.51</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Note: this adds a test dependency in the jclouds project pom (the one that is used as a parent project). The test dependency might be used in multiple providers/apis live tests...